package/src/languageFeatures.ts

```diff
@@ -652,6 +652,22 @@ function toTextEdit(textEdit: ls.TextEdit): monaco.editor.ISingleEditOperation {
     };
 }
 
+const DOCS_BASE_URL = "https://learn.microsoft.com/azure/data-explorer/kusto/query";
```

Comment:
shouldn't the base url be configurable?

Reply:
Creating a task to make also the actual links configurable